src/main/react/app/containers/App/constants.js

```diff
@@ -51,17 +51,18 @@ export const getDatePresets = (type) => {
   }
 };
 
+export const PLACEHOLDER_PRESET_NOT_SELECTED = "Empty"
 
 export const GREGORIAN_DATE_PRESETS = [
```

Comment:
Can you add a Jira task to figure out a way to load presets from storage? I think we probably will eventually want a way to extend this without modifying this javascript code

Reply:
Right, we should add one for that.